var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -298,6 +296,22 @@ def on_off(varstr):
             else:
                 args.append('-DSUPERLUMT_THREAD_TYPE=Pthread')
 
+        # Building with SuperLU_DIST
+        if '+superlu-dist' in spec:
+            args.extend([
+                '-DSUPERLUDIST_ENABLE=ON',
+                '-DSUPERLUDIST_INCLUDE_DIR=%s'
+                % spec['superlu-dist'].prefix.include,
+                '-DSUPERLUDIST_LIBRARY_DIR=%s'
+                % spec['superlu-dist'].prefix.lib,
+                '-DSUPERLUDIST_LIBRARIES=%s'
+                % spec['blas'].libs
+            ])
+            if spec.satisfies('+openmp'):
+                args.extend([
+                '-DSUPERLUDIST_OpenMP=ON'
```

Comment:
Has the SuperLU_DIST spack package been updated in have a `+openmp` variant?

Reply:
No it has not, but it builds with OpenMP by default.